scripts/o3de/ExportScripts/export_standalone_monolithic_engine_centric.py

```diff
@@ -0,0 +1,160 @@
+                        help="Format of archive to create from the output directory",
+                        choices=["zip", "gzip", "bz2", "xz"], default="zip")
+parser.add_argument('-sl', '--seedlist', type=pathlib.Path, dest='seedlist_paths', action='append',
+                        help='Path to a seed list file for asset bundling. Specify multiple times for each seed list.')
+parser.add_argument('-gf', '--gem-folder-name-to-copy', type=str, dest='gem_folder_names_to_copy', action='append',
+                        help="The name of a Gem folder to include in release. Only specify folder name. This should match what is in the project directory.")
+parser.add_argument('-pfp', '--project-file_pattern-to-copy', type=str, dest='project_file_patterns_to_copy', action='append',
+                        help="Any additional file patterns located in the project directory. File patterns will be relative to the project path.")
+parser.add_argument('-bnmt', '--build-non-mono-tools', action='store_true')
+parser.add_argument('-nmbp', '--non-mono-build-path', type=pathlib.Path, default=None)
+parser.add_argument('-mbp', '--mono-build-path', type=pathlib.Path, default=None)
+parser.add_argument('-nogame', '--no-game-launcher', action='store_true', help='this flag skips building the Game Launcher on a platform if not needed.')
+parser.add_argument('-noserver', '--no-server-launcher', action='store_true', help='this flag skips building the Server Launcher on a platform if not needed.')
+parser.add_argument('-nounified', '--no-unified-launcher', action='store_true', help='this flag skips building the Unified Launcher on a platform if not needed.')
+parser.add_argument('-pl', '--platform', type=str, default=host_asset_platform, choices=['pc', 'linux', 'mac'])
+parser.add_argument('-a', '--archive-output', action='store_true', help='This option places the final output of the build into a compressed archive')
+parser.add_argument('-q', '--quiet', action='store_true', help='Suppresses logging information unless an error occurs.')
+
+args = parser.parse_args(o3de_context.args)
+
+if args.quiet:
+    o3de_logger.setLevel(logging.ERROR)
+
+non_mono_build_path = (o3de_context.engine_path) / 'build' / 'non_mono' if args.non_mono_build_path is None else args.non_mono_build_path
+mono_build_path     = (o3de_context.engine_path) / 'build' / 'mono' if args.mono_build_path is None else args.mono_build_path
+
+selected_platform = args.platform
+
+if not selected_platform:
+    o3de_logger.error("Unable to identify default host platform! Please supply the platform using '--platform'. Options are [pc, linux, mac].")
+    sys.exit(1)
+
+o3de_logger.info(f"Selected Project path: {o3de_context.project_path}")
+o3de_logger.info(f"Selected Engine path : {o3de_context.engine_path}")
+o3de_logger.info(f"Build path for non-monolithic executables: {args.non_mono_build_path}")
+o3de_logger.info(f"Build path for monolithic executables: {args.mono_build_path}")
```

Comment:
nit: Lets line up the paths for better reading
```suggestion
o3de_logger.info(f"Selected Project path                     : {o3de_context.project_path}")
o3de_logger.info(f"Selected Engine path                      : {o3de_context.engine_path}")
o3de_logger.info(f"Build path for non-monolithic executables : {args.non_mono_build_path}")
o3de_logger.info(f"Build path for monolithic executables     : {args.mono_build_path}")
```